mariner-installer/build/build

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -ex
+
+echo "BRANCH: $BRANCH"
+cd $WORKSPACE/mariner-installer
+
+. /etc/os-release
```

Comment:
My bad! I thought that CentOS 6 had `/etc/os-release`, and I see now that [it does not] (https://bugs.centos.org/view.php?id=8894). Let's just go with ceph-deploy's `listcontains()`